.eslintrc.json

```diff
@@ -2,6 +2,7 @@
   "env": {
     "browser": true,
     "es2021": true,
+    "node": true,
```

Comment:
I think this line was added when trying the Node's package of Mixpanel, and then was leave there. Please remove it if that was the case.

Reply:
I remembered why I added this. It was to fix "process is undefined" error when using process.env